airbyte-container-orchestrator/Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG JDK_VERSION=14.0.2
+FROM openjdk:${JDK_VERSION}-slim AS sync-attempt
+
+ARG DOCKER_BUILD_ARCH=amd64
```

Comment:
will this cause an issue on non-m1?

Reply:
This default is `amd64` not arm. We'll only override on M1. This is the existing pattern that was in use for the scheduler.